app/services/mlops/model_deployment_service.py

```
@@ -0,0 +1,711 @@
+            alerts = []
+            async for doc in cursor:
+                alerts.append(doc)
+
+            return alerts
+
+        except Exception as e:
+            logger.error(f"Error getting recent alerts: {e}")
+            return []
+
+    async def _get_previous_stable_version(
+        self, model_id: str, current_version: str
```

Ruff (0.13.2)

Unused method argument: `model_id`

(ARG002)